provider/openstack/firewaller.go

```diff
@@ -36,7 +36,8 @@ var extractControllerRe = regexp.MustCompile(GroupControllerPattern)
 var shortRetryStrategy = retry.CallArgs{
 	Clock:       clock.WallClock,
 	MaxDuration: 5 * time.Second,
-	Delay:       200 * time.Millisecond,
+	Delay:       3 * time.Second,
```

Comment:
This is very long considering we're now using backoff.
I'd also use `retry.ExpBackoff` with jitter configured. And a min delay of 1s and max delay of say 5s. The backoff factor can be 1.5 (2 is a bit high)